torch/_dynamo/variables/torch.py

```diff
@@ -154,14 +154,31 @@ def __init__(self, value, **kwargs):
         self.value = value
 
     def reconstruct(self, codegen):
-        return torch_reconstruct(codegen, self.value)
+        name = torch_get_name(value, f"allowed_fn_{id(value)}")
+        unique_var_name = "__" + re.sub(r"[^a-zA-Z0-9_]+", "_", name)
+        return codegen.setup_globally_cached(unique_var_name, value, False)
```

Comment:
This seems exactly the same as `torch_reconstruct`, why did you inline it?

Reply:
Oh, because we have the ```BaseTorchVariable```, so put this in the base class and removed the original global helper function.

Reply:
But you haven't deleted the helper function

Reply:
Resolved!